packages/subgraph-service/contracts/DisputeManager.sol

```diff
@@ -196,6 +196,45 @@ contract DisputeManager is
         return (dId1, dId2);
     }
 
+    /// @inheritdoc IDisputeManager
+    function createLegacyDispute(
```

Comment:
nit: might be better to call this `slashLegacyDispute` or `createAndAcceptLegacyDispute` to make it clearer that it's not just creating it but also settling it?

Reply:
changed to `createAndAcceptLegacyDispute`